spinn_front_end_common/utilities/database/database_writer.py

```diff
@@ -140,6 +141,9 @@ def add_application_vertices(self):
         with self.transaction() as cur:
             # add vertices
             for vertex in FecDataView.iterate_vertices():
+                max_atoms = vertex.get_max_atoms_per_core()
+                if not isinstance(max_atoms, int):
```

Comment:
if type conversion is needed I would put it in the vertex so it works everywhere.

Reply:
These lines have been deleted, but the functionality of the maximum atoms per core has been refactored to account for this anyway